source/slang-capture-replay/slang-global-session.cpp

```diff
@@ -1,5 +1,5 @@
-
-#include <vector>
+#include <sstream>
+#include <thread>
```

Comment:
Are these needed?

Reply:
No need anymore. I remove some the logic into capture-manager.cpp

Reply:
I remove those in the latest update.